en/core-libraries/httpclient.rst

```diff
@@ -479,6 +479,60 @@ instead. You can force select a transport adapter using a constructor option::
 
     $client = new Client(['adapter' => Stream::class]);
 
+Events
+======
+
+The HTTP client triggers couple of events before and after sending a request
+which allows you to modify either the request or response or do other tasks like
+caching, logging etc. You can use the global event manager to setup listeners
+for these events.
+
+HttpClient.beforeSend
+---------------------
+
+    // Somewhere before calling one of the HTTP client's send methods.
+    \Cake\Event\EventManager::instance()->on(
```

Comment:
Aren't events only dispatched on the client object and not on the global event manager?

Reply:
Hmm.. This makes me think that suggesting `$client->getEventManager()->on()` would be better.